source/open_cdm_adapter.cpp

```diff
@@ -45,6 +42,23 @@ OpenCDMError opencdm_gstreamer_session_decrypt(struct OpenCDMSession *session, G
                                                 nullptr);
 }
 
+OpenCDMError opencdm_gstreamer_session_decrypt_buffer(struct OpenCDMSession *session, GstBuffer *buffer, GstCaps *caps)
```

Comment:
Will this compile in all build environments when opencdm_gstreamer_session_decrypt_buffer() isn't declared in open_cdm_adapter.h? Or will we need some conditional compilation?

Reply:
I didn't think it should matter. Maybe its safer to compile out as we may get some warnings as errors because its not in the header and that will fail the compilation.